### ISSUE TITLE
Improve netifyd network configuration

### ISSUE THREAD

**Issue by gsanchietti** [NethServer/nethsecurity#929]
In accordance with Netifyd guidelines, modify the Netifyd configuration file, to limit the number of interfaces to inspect.

Actually every local interface (physical or logical) is added netifyd `internal_if` parameter.
Keeping both physical and VLAN interfaces introduces redundancy and has a few negative impacts:
- Wasted memory and CPU: A new thread is created for each interface, consuming more memory and CPU time to process duplicated flows.
- Flow duplication: Since the VLAN ID is part of the flow hashing algorithm, the same flow appears differently on both physical and VLAN interfaces, leading to duplicates in memory. This could result in inaccurate or unexpected flow processing. Best practice suggests listening only on the physical interface. 

The following interfaces must be always be added to netifyd configuration:

* [x] Physical Interfaces: Yes
* [x] VLAN: No, just the underlying physical device 
* [x] Bond: Direct bond interface
* [x] Bridge: Direct bridge interface
 
Currentlyalso VPNs and hotspots interfaces are added to netifyd configuration, but there are mixed feelings about having them in netifyd configuration.

* somebody doesn't like to have them cause it means more cpu load to analyze flows
* some other would like to have them so that we can see and record vpn traffic in the monitoring (e.g. a rw host is using NethSecurity as its gateway and it is doing a lot of traffic, we wouldn't see it at all if vpn interfaces are not managed) .

Proposed changes:
- keep current logic as default for VPN and Hotspot
- [x] add an option to exclude some devices, the option could be a list with glob matching, like: `eth1`, `tun*`, `wg*`.
- [x] do not expose the option inside the UI, but just document how to configure it

**Comment by gsanchietti** [comment]
Testing image: 23.05.5-ns.1.3.0-92-g7ef1d292

**Comment by gsanchietti** [comment]
Before adding the exclusion:
```
# uci show netifyd
netifyd.@netifyd[0]=netifyd
netifyd.@netifyd[0].enabled='1'
netifyd.@netifyd[0].autoconfig='0'
netifyd.@netifyd[0].exclude='tun*'
netifyd.@netifyd[0].internal_if='br-lan' 'eth0' 'tunrw1'
netifyd.@netifyd[0].external_if='eth1'
```

From `ps` command:
```
root     13954  0.1  1.1  15548 11936 ?        Sl   13:02   0:00 /usr/sbin/netifyd -R -I br-lan -I eth0 -I tunrw1 -E eth1
```

After added exclusion for `tun*`:
```
# uci show netifyd
netifyd.@netifyd[0]=netifyd
netifyd.@netifyd[0].enabled='1'
netifyd.@netifyd[0].autoconfig='0'
netifyd.@netifyd[0].ns_exclude='tun*'
netifyd.@netifyd[0].internal_if='eth0' 'br-lan'
netifyd.@netifyd[0].external_if='eth1'
```

From `ps` command:
```
root     14216  0.0  1.5  27456 16112 ?        Sl   13:04   0:00 /usr/sbin/netifyd -R -I eth0 -I br-lan -E eth1
```

**Comment by gsanchietti** [comment]
Released on 23.05.5-ns.1.4.0
